contents/docs/product-analytics/llms.md

```diff
@@ -0,0 +1,45 @@
+---
+title: LLM insights (beta)
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+> This integration is currently in private beta. If you'd like to be added to the beta, email Lior@posthog.com.
+
+We've teamed up with [Langfuse](https://langfuse.com/docs/analytics/posthog) to track metrics for LLM applications. Langfuse is an open source solution for monitoring LLMs. It tracks metrics such as model costs, latency, token usage, and more.
+
+Combining your Langfuse and PostHog data makes it easy to answer questions like:
+
+- What are my LLM costs by customer, model and in total?
+- How many of my users are interacting with my LLM features?
+- Are there latency spikes in my LLMs?
+- Does interacting with LLM features correlate with other metrics (retention, usage, revenue etc.)?
+
+Here's an [example dashboard in PostHog](https://eu.posthog.com/shared/HPOaK5zNVkP062nQJQJoooXe61l15w):
+
+![LLM analytics example dashboard](../../images/docs/product-analytics/llm-example-dash.mp4)
```

Comment:
This is broken in the preview, I think it is pointing to the wrong folder location? Might just be `../images/docs/product-analytics/llm-example-dash.mp4`